_studio/mfx_lib/shared/src/libmfxsw_pak.cpp

```diff
@@ -141,13 +141,6 @@ mfxStatus MFXVideoPAK_Init(mfxSession session, mfxVideoParam *par)
     MFX_CHECK(par, MFX_ERR_NULL_PTR);
     try
     {
-        // close the existing PAK unit,
-        // if it is initialized.
-        if (session->m_pPAK.get())
-        {
-            MFXVideoPAK_Close(session);
-        }
-
         // create a new instance
```

Comment:
@mgonchar : we need your input here: is that save?

Reply:
Not sure, the difference is only in call of
// wait until all tasks are processed
        session->m_pScheduler->WaitForTaskCompletion(session->m_pPAK.get());

Without this piece of code that call won't be performed. Cleanup of allocated resources by PAK interface should be correctly performed by destructor

I believe it looks reasonable to have this call, but I'm not really familiar with scheduler code